### PR TITLE
feat: nav-bar added.

### PR DESCRIPTION
**Pull request by krishnakant01**
I've added a nav bar to my e-commerce website to get started.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          </a>
+
+          <a href="#">
+            <li class="nav-link-list-item">Football Kits</li>
+          </a>
+        </ul>
+      </div>
+
+      <div class="nav-social-links">
+        <ul class="">
```
Comment (tushartiwari7) [inline index.html]: Remove empty classes

File: index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          </li>
+
+          <li class="social-link-list-items">
+            <a href="#">
+              <i class="material-icons">shopping_bag</i>
+            </a>
+          </li>
+
+          <li class="social-link-list-items">
+            <!-- <a href="#" class="btn btn-link-primary padding-size-md">LOG IN</a> -->
```
Comment (tushartiwari7) [inline index.html]: Remove comments

**Review by tushartiwari7 — APPROVED** [review]
Please remove dead code and you're good to merge.

**Review by Manju-sree — APPROVED** [review]
apart from comments above.. rest code looks good to merge

**Pull request merged**